### ISSUE TITLE
Docs feedback: The topic local.file_match lacks detail on structure for path_targets.

### ISSUE THREAD

**Issue by clayton-cornell** [grafana/alloy#736]
### URL

https://grafana.com/docs/alloy/latest/reference/components/local.file_match/

### Feedback

The topic https://grafana.com/docs/alloy/latest/reference/components/local.file_match/ lacks detail on structure for path_targets.  The doublestar doc https://github.com/bmatcuk/doublestar doesn't provide relevant information.

The example at https://grafana.com/docs/grafana-cloud/monitor-infrastructure/integrations/integration-reference/integration-apache-http/  uses undocumented features.

```
local.file_match "logs_integrations_integrations_apache_access" {
	path_targets = [{
		__address__ = "localhost",
		__path__    = "/var/log/apache2/access.log",
		instance    = constants.hostname,
		job         = "integrations/apache_http",
	}]
}
```

**Comment by Konafets** [comment]
The documentation is still missing those information.

**Comment by clayton-cornell** [comment]
Yes, you're right. There's quite a few of these in the doc backlog. I'm working through the backlog as I can. The issue hasn't been forgotten.

**Comment by mikecomer-beakon** [comment]
+1 to document this

**Comment by Depicus** [comment]
This still seems outstanding

**Comment by clayton-cornell** [comment]
Yes. It's still in my queue though. It hasn't been forgotten.
